rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - base
+    ztrigger:
+      - pr:
+          CRON: ""
+      - periodic:
+          branches: "do_not_build_on_pr"
+          NUM_TO_KEEP: 10
+    jobs:
+      - 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    STAGES: >
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Run IRR Tests,
+      Cleanup,
+      Destroy Slave
+    branch: master
+    NUM_TO_KEEP: 30
+
+    # TEMPLATE
+    name: 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      # IRR REPOS ARE ADDED HERE
+      - github:
+          url: https://github.com/rcbops/rpc-maas
+    parameters:
+      - rpc_gating_params
+      - single_use_slave_params:
+         IMAGE: "{IMAGE}"
+         FLAVOR: "{FLAVOR}"
+         REGION: "{REGION}"
+         IRR_CONTEXT: "{context}"
```

Comment:
The intent here was to provide the context, image and series to the environment the test runs in so that run_tests.sh could perhaps do things based on their values. If there's a better way to do this (without making them editable params on the job) then please let us know how.